store/redis/tenant.go

```diff
@@ -61,30 +61,52 @@ type TenantCache interface {
 }
 
 type tenantCache struct {
-	clients map[uint64]*dbdrivers.RedisDBConn
-	prefix  string
+	clients   map[uint64]*dbdrivers.RedisDBConn // map[tenantID]*redis.Client
+	prefix    string                            // prefix for all keys cocatenated with underscore "_"
+	operation string                            // operation name for tracing
 }
 
 // NewTenantCache creates a new TenantCache if you enable tenant mode (`detabase.tenant.on` in config).
 // This assumes it is called after the (*Bean).InitDB() func and takes (bean.DBDeps).TenantRedisDBs as input.
-func NewTenantCache(tenants map[uint64]*dbdrivers.RedisDBConn, prefix string) TenantCache {
+func NewTenantCache(tenants map[uint64]*dbdrivers.RedisDBConn, prefix string, opts ...TenantCacheOption) TenantCache {
 
-	return &tenantCache{
+	t := &tenantCache{
 		clients: tenants,
 		prefix:  prefix,
 	}
+
+	for _, opt := range opts {
+		opt(t)
+	}
+
+	return t
+}
+
+type TenantCacheOption func(*tenantCache)
+
+// OptTraceTC is an option that enables tracing for all redis operations in TenantCache with the given operation name
+// if you enable Sentry's sampling for traces in cofing.
+// The operation name will be `tenant-cache` by default if it is passed an empty string.
+func OptTraceTC(operation string) func(*tenantCache) {
+	return func(t *tenantCache) {
+		if operation != "" {
+			t.operation = operation
+		} else {
+			t.operation = "tenant-cache" // by default
```

Comment:
should put this default operation value to `NewTenantCache`.  because if user doesn't transport this method, then the operation will be empty.